src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/HistogramAdvice.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace System.Diagnostics.Metrics
+{
+    /// <summary>
+    /// Contains settings used to advise metrics consumers how to construct storage for <see cref="Histogram{T}"/> instruments.
+    /// </summary>
+    /// <typeparam name="T">Histogram value type.</typeparam>
+    public sealed class HistogramAdvice<T> where T : struct
+    {
+        /// <summary>
+        /// Constructs a new instance of <see cref="HistogramAdvice{T}"/>.
+        /// </summary>
+        /// <param name="explicitBucketBoundaries">
+        /// <para>Explicit bucket boundaries advised to be used with the histogram.</para>
+        /// <para>Notes:
```

Comment:
><para>Notes: [](http://example.com/codeflow?start=12&length=12)

Moving this to <remarks> element will be better and consistent to the other docs.

Reply:
Added remarks on the property too.

Reply:
Will you move this notes section to a remark's element instead?

Reply:
Updated